client/src/app/layout/main-content/shared-style.scss

```diff
@@ -47,7 +47,7 @@
     }
 
     .recipe-cards-container{
-      width:100%;
+      width:97%;
```

Comment:
😢

Reply:
It's prettier that way 🦓

Reply:
Please try and make it prettier in a more stable way. What you're doing right now is an anti-pattern in CSS called "magic numbers".

Reply:
https://github.com/RumyanaA/Recipe-finder/pull/21/commits/111c1f3e761bf7e5990b67acb789ce8ddabaaf7e